crypto: add extensions property to X509Certificate (#48780)
nodejs/node: doc/api/crypto.md

Verifies that this certificate was signed by the given public key.
Does not perform any other validation checks on the certificate.

### `x509.extensions`

> The `X509Certificate` class provides access to various
> properties of X.509 certificates, including extensions.
> Here, we describe the `x509.extensions` property,
> which represents the extensions of the certificate.
> 
> ### x509.extensions
> 
> * Type: {Object}
> 
> The `x509.extensions` property returns an object representing
> the extensions of the certificate. These extensions
> provide additional information about the certificate.
> 
> Example:

Reviewer: ```suggestion

<!-- YAML
added: REPLACEME
-->

The `X509Certificate` class provides access to various
properties of X.509 certificates, including extensions.
Here, we describe the `x509.extensions` property,
which represents the extensions of the certificate.

### x509.extensions

Returns: {Object}

Returns an object representing the extensions of the certificate.

Example: Retrieving X.509 Certificate extensions
```

> The `X509Certificate` class provides access to various
> properties of X.509 certificates, including extensions.
> Here, we describe the `x509.extensions` property,
> which represents the extensions of the certificate.
> 
> ### x509.extensions
> 
> * Type: {Object}
> 
> The `x509.extensions` property returns an object representing
> the extensions of the certificate. These extensions
> provide additional information about the certificate.
> 
> Example:
> 
> ```cjs
> const { X509Certificate } = require('crypto');
> 
> const certPem = '...'; // Replace with your X.509 certificate in PEM format
> const x509 = new X509Certificate(certPem);
> 
> const extensions = x509.extensions;
> console.log(extensions);
> ```

Reviewer: I don't think the example adds any value, so I suggest to remove it.

I'm suggesting to add a CHANGED entry, fix the way Return value is documented and removed the unnecessary prose all around.

```suggestion

<!-- YAML
added: REPLACEME
-->

The `X509Certificate` class provides access to various
properties of X.509 certificates, including extensions.
Here, we describe the `x509.extensions` property,
which represents the extensions of the certificate.

### x509.extensions

Returns: {Object}

Returns an object representing the extensions of the certificate.
```
